alica_engine/include/engine/blackboard/Blackboard.h

```diff
@@ -104,20 +386,22 @@ class LockedBlackboardRW
     LockedBlackboardRW& operator&=(LockedBlackboardRW&) = delete;
     LockedBlackboardRW(LockedBlackboardRW&) = delete;
 
-    template <class... Args>
-    [[deprecated]] void registerValue(const std::string& key, Args&&... args)
+    template <typename T>
+    const T& get(const std::string& key) const
     {
-        _impl->registerValue(key, std::forward<Args>(args)...);
+        return _impl->get<T>(key);
     }
-
     template <typename T>
     T& get(const std::string& key)
     {
         return _impl->get<T>(key);
     }
```

Comment:
technically one can be implemented, and other could be made with const_cast. But I guess this is also OK

Reply:
actually, it is better this way since this is just a wrapper class, so it should just forward the call